CompanyEmployees.Application/Validators/CheckInAttendanceCommandValidator.cs

```diff
@@ -0,0 +1,37 @@
+using CompanyEmployees.Application.Commands;
+using FluentValidation;
+using FluentValidation.Results;
+
+namespace CompanyEmployees.Application.Validators;
+
+public sealed class CheckInAttendanceCommandValidator : AbstractValidator<CheckInAttendanceCommand>
+{
+    public CheckInAttendanceCommandValidator()
+    {
+        RuleFor(a => a.Attendance.EmployeeId)
+            .NotEmpty()
+            .WithMessage("Employee ID is required.");
+
+        RuleFor(a => a.Attendance.CheckInTime)
+            .NotEmpty()
+            .WithMessage("Check-in time is required.");
+
+        RuleFor(a => a.Attendance.Status)
+            .NotEmpty()
```

Comment:
The combined use of `.NotEmpty()` and `.MaximumLength(10)` with a single `.WithMessage(...)` may introduce ambiguity in which rule caused the failure. Consider separating them with individual messages.

**Suggested change**
```suggestion
RuleFor(a => a.Attendance.Status)
    .NotEmpty().WithMessage("Status is required.")
    .MaximumLength(10).WithMessage("Status must not exceed 10 characters.");
```